### ISSUE TITLE
[12.0/13.0] Stock create method code with strange structure

### ISSUE THREAD

**Issue by reinaldoramosarxi** [odoo/odoo#41813]
Hello,

While looking over some code in the stock module, we came across something in the create method (v12.0, but it still has the same issue in v13.0) that doesn't make a lot of sense to us.

The following line calls a vals dict, which in that context takes the value of the last iteration (unless we're missing something). The create method receives a vals_list, which is used in the previous for cycle.

https://github.com/odoo/odoo/blob/13.0/addons/stock/models/stock_move_line.py#L206

So basically, we're asking if the iteration is correct (maybe an explanation on what it is supposed to be doing) or if something was overlooked and it's really a bug.

Thank you.

**Comment by Feyensv** [comment]
Indeed, it seems to be a leftover of a migration from `create` to `create_multi`.

Cc: @sle-odoo
